.prettierrc.cjs

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  useTabs: false,
+  singleQuote: true,
+  semicolons: false,
```

Comment:
I think this is supposed to be `semi` instead of `semicolons`.

Reply:
Dang, can't believe I missed; fixed in latest commit.